gulpfile.js

```diff
@@ -327,17 +327,17 @@ gulp.task('manifest', 'Generate a browser-specific manifest file.', (done) => {
 
 
 gulp.task('pre-commit-lint-js', () => {
-    return guppy.stream('pre-commit')
-        .pipe(filter(['*.js']))
+    return gulp.src(guppy.src('pre-commit'))
```

Comment:
I tried various things to get it working again, and with this is worked. But looking at the documentation I think `guppy.stream` is what we want not `guppy.src`:

> Note that depending on the git-hook, you may be acting on files that differ from your working copy, such as those staged for commit (as with 'pre-commit' for example), rather than the working copy. If you need to act on the working-copy files, use guppy.src() instead.

I'll check and see if the stream works now, maybe it was just the filter that was too narrow.